modules/library/jdbc/src/main/java/org/geotools/data/jdbc/FilterToSQL.java

```diff
@@ -531,6 +521,21 @@ public Object visit(PropertyIsLike filter, Object extraData) {
         return extraData;
     }
 
+    protected void processLeftOperand(Object extraData, boolean matchCase, Expression att, Class attributeType)
```

Comment:
Poor name, process left operand of what?

```suggestion
    protected void processLikeLeftOperand(Object extraData, boolean matchCase, Expression att, Class attributeType)
```